### DISCUSSION TITLE
ioSender

### DISCUSSION THREAD

**Discussion by Jerryl50** [terjeio/ioSender#340] [Q&A]
When I connect to my cnc router using ioSender or gSender they say they are connected (via usb) but neither program will function or control the cnc. Any tips this is my first build. I have assembled an Outback CNC by Maker Store.

**Comment by phil-barrett** [comment]
What controller do you have?

**Reply by Jerryl50** [reply]
An E5X MCS T4.1 ethernet 5 axis motion controller and a Folinn BD600 Series VSD

**Reply by phil-barrett** [reply]
Have you talked to makerstore?  They should get you going.

**Comment by Jerryl50** [comment]
I had not installed the CP2102 USB drivers. I have installed them now, and will see if it is working now.
